news/2025/2025-01-27-performance-points-star-rating-updates.md

```diff
@@ -0,0 +1,224 @@
+
+The combination of effective BPM and density evaluation ensures that fast, spaced-out sections feel accurately represented in the difficulty system while avoiding inflation for simpler sections at high BPM:
+
+![](/wiki/shared/news/2025-01-27-performance-points-star-rating-updates/reading-graph.png)
+
+As part of this change, fixed HR buffs in performance calculations were removed to allow difficulty calculation to handle the difficulty increase.
+
+### Ratio considerations in colour
+
+A [change](https://github.com/ppy/osu/pull/31285) proposed by [ltca](https://osu.ppy.sh/users/11475208) has been created in order to consider the ratios of patterns when awarding colour difficulty.
+
+The colour skill now applies a penalty based on the number of consecutive *consistent* intervals, meaning stream-like patterns with a lack of rhythmic and colour difficulty are penalised appropriately. This also means that diverse beatmaps with less predictable colour changes receive a buff.
+
+Some examples of beatmaps buffed by this change:
+
+- [Babuchan - 13 stairs [Unneeded]](https://osu.ppy.sh/beatmapsets/1093671#taiko/3878430): 9.43* --> 10.00*
+- [GTS Sound Team - <</nttld.:beings>> \~Truth in Uncertainty\~ [<</eclectic.:genesis>>]](https://osu.ppy.sh/beatmapsets/1859338#taiko/3822143): 9.10* --> 9.79*
+- [Ludicin - Fallen Symphony [The Symphony of the Fallen Angel]](https://osu.ppy.sh/beatmapsets/1957129#taiko/4054390): 8.90* --> 9.60*
+
+Some examples of beatmaps nerfed by this change:
+
+- [Null Specification - Aletheia (fake lover, fake summer) [fake promise]](https://osu.ppy.sh/beatmapsets/1895850#taiko/3907074): 7.47* --> 6.82*
+- [DJ SHARPNEL - Pacific Girls [Divine Orchestra]](https://osu.ppy.sh/beatmapsets/2176923#taiko/4597047): 7.37* --> 6.68*
+- [USAO - Try Again [Inner Oni]](https://osu.ppy.sh/beatmapsets/1875585#taiko/3859766) with DT: 7.61* --> 6.83*
+
+### Stamina improvements
+
+A [change](https://github.com/ppy/osu/pull/31337) proposed by [ltca](https://osu.ppy.sh/users/11475208) has been created in order to better factor speed into stamina calculations:
+
+- Monos now have an additional buff to acknowledge the stamina required to keep up with rapid single-colour patterns, especially at higher BPMs
+- Stamina now considers the density and BPM of consecutive objects, awarding extra bonuses for faster patterns
+
+### Convert changes
+
+Star rating for convert beatmaps have been improved, thanks to a [set](https://github.com/ppy/osu/pull/31196) of [changes](https://github.com/ppy/osu/pull/31546) proposed by [ltca](https://osu.ppy.sh/users/11475208). There is no longer any blanket star rating nerf for converts, because they're instead now more properly addressed in various areas:
+
+- Low accuracy is scaled more harshly on beatmaps with a lot of mono patterns
```

Comment:
we're talking about a nerf and not a buff, right? "scale" is vague in this sense (and doesn't mean much without the mathematical context anyway)

```suggestion
- Low accuracy is punished more harshly on beatmaps with a lot of mono patterns
```